_layouts/default.html

```diff
@@ -191,7 +203,7 @@
         <div id="mainhome" class="main-content">
           <div id="content">
             <nav class="navbar navbar-expand-lg navbar-light bg-light container-fluid">
-                <button type="button" id="sidebarCollapse" class="btn btn-info new mr-3">
+                <button class="button" type="button" id="sidebarCollapse" class="btn btn-info new mr-3">
```

Comment:
There is already a Class Attribute Mentioned in the Button Tag, Please do add this button class in it. Also please add the before and after screenshots to show the difference.